src/monitoring_service/database.py

```diff
@@ -1,9 +1,9 @@
+import dataclasses
```

Comment:
Don't know why this happens, seems to be a problem on travis. But for now you have to revert  this in order to make the linting check pass.